README.md

```diff
@@ -49,10 +55,10 @@ steps:
 
 - `location`: (Required) Location (Region/Zone) for the cluster.
 
-- `credentials`: (Optional) Service account key to use for authentication. This should be
+- `credentials`: (**Deprecated**) This input is deprecated. See [auth section](https://github.com/google-github-actions/get-gke-credentials#via-google-github-actionsauth) for more details.
```

Comment:
Can we move this to the very end of the list?

Reply:
done